ImmuDB4Net/ImmuClient.cs

```diff
@@ -206,7 +322,8 @@ public async Task Open(string username, string password, string defaultdb)
                 Address = GrpcAddress,
                 ShutdownTimeout = ConnectionShutdownTimeout
             });
-            ActiveSession = await SessionManager.OpenSessionAsync(Connection, username, password, defaultdb);
+            ActiveSession = await SessionManager.OpenSessionAsync(Connection, username, password, databaseName);
+            ValidateLocalState();
             heartbeatCloseRequested = new ManualResetEvent(false);
```

Comment:
we don't set the value of `currentDb` here.

Reply:
oops, fixed.